### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MattKingDS**

### CODE REVIEW

**Comment by MattKingDS** [comment]
Uploading the new C# Readme

File: README.md [hunk 1]
```diff
@@ -1,161 +1,95 @@
...
+6. Click **Install**.
 
-Copy `DocuSign.eSign.dll` file to your local machine and add reference through your project settings.
+### Package Manager Console:
 
-### Others
+1. Open or create a new C# project.
+2. Open the **Package Manager Console** by either:
+  1. Clicking **Package Manager Console** along the bottom of Visual Studio
+  2. Clicking **Tools** -\&gt; **NuGet Package Manager** -\&gt; **Package Manager Console**
```
Comment (RajRele) [inline README.md]: Check this please.

File: README.md [hunk 2]
```diff
@@ -1,161 +1,95 @@
...
-* BouncyCastle
-* System.IdentityModel.Tokens.Jwt
+This client has the following external dependencies:
 
-Usage
-=====
+### .NET Standard v2 and .NET Core:
 
-To send a signature request from a template using OAuth JWT authentication:
+- CSharp v4.5.0
```
Comment (RajRele) [inline README.md]: Should be Microsoft.CSharp

File: README.md (hunk 1 above)
Comment (MattKingDS) [reply README.md]: This is technically correct. Removed URL encoding.

File: README.md (hunk 2 above)
Comment (MattKingDS) [reply README.md]: Updated / alphabetized the Dependency list

**Pull request merged**